### ISSUE TITLE
chemistry - create spanish .yml file

### ISSUE THREAD

**Issue by lwtchu** [openstax/kitchen#237]
Do after #234 - since this one is also done and needs a Spanish file, we can get started on this. Low priority until the content comes back to us at some point.

**Comment by lwtchu** [comment]
Note: need to update Chemistry recipe to latest version o f kitchen so it will work with i18n
